test/powershell/Modules/Microsoft.PowerShell.Utility/FormatHex.Tests.ps1

```diff
@@ -480,14 +428,15 @@ Describe "FormatHex" -tags "CI" {
             $result.ToString() | Should be "00000000   61 61 61 61 61 61 61 61 61 61 61 61 61 61 61 61  aaaaaaaaaaaaaaaa`r`n00000010   61 61 61 61 61 61 61 61 61 61 61 61 61 61        aaaaaaaaaaaaaa  "
         }
 
-        It "Validate that files do not have buffer underrun problems 'Format-Hex -path `$InputFile2'" {
+        It "Validate that files do not have buffer underrun problems 'Format-Hex -path `$InputFile4'" {
 
-            $result = Format-Hex -path $InputFile2
+            $result = Format-Hex -path $InputFile4
 
             $result | Should Not BeNullOrEmpty
-            $result.Count | should be 2
-            $result[0].ToString() | Should be "00000000   54 68 69 73 20 69 73 20 61 20 62 69 74 20 6D 6F  This is a bit mo"
-            $result[1].ToString() | Should be "00000010   72 65 20 74 65 78 74                             re text         "
+            $result.Count | Should Be 3
+            $result[0].ToString() | Should be "00000000   4E 6F 77 20 69 73 20 74 68 65 20 77 69 6E 74 65  Now is the winte"
+            $result[1].ToString() | Should be "00000010   72 20 6F 66 20 6F 75 72 20 64 69 73 63 6F 6E 74  r of our discont"
```

Comment:
Should use MatchExactly like here: https://github.com/PowerShell/PowerShell/blob/7b8c6e7f548e5ab98e9d36d073e5af7d989c8f79/test/powershell/Modules/Microsoft.PowerShell.Management/FileSystem.Tests.ps1#L499